headers/fmi3FunctionTypes.h

```diff
@@ -453,20 +528,19 @@ typedef fmi3Status fmi3GetOutputDerivativesTYPE(fmi3Instance instance,
 typedef fmi3Status fmi3DoStepTYPE(fmi3Instance instance,
                                   fmi3Float64 currentCommunicationPoint,
                                   fmi3Float64 communicationStepSize,
-                                  fmi3Boolean noSetFMUStatePriorToCurrentPoint);
+                                  fmi3Boolean noSetFMUStatePriorToCurrentPoint,
+                                  fmi3Boolean* earlyReturn);
```

Comment:
The value of `earlyReturn` can be derived from `fmi3IntermediateUpdateInfo.intermediateUpdateTime` provided by the callback function, so it should be omitted here.

Reply:
We cannot rely on the == to work on two float times. This might be the reason for the flag.